Dockerfile

```diff
@@ -1,32 +1,33 @@
 ARG venv_python
-ARG alpine_version
-FROM python:${venv_python}-alpine${alpine_version}
+FROM python:${venv_python}
 
 LABEL Maintainer="CanDIG Project"
 LABEL "candigv2"="opa"
 
 USER root
 
-RUN addgroup -S candig && adduser -S candig -G candig
+RUN groupadd -r candig && useradd -rm candig -g candig
 
-RUN apk update
-
-RUN apk add --no-cache \
+RUN apt-get update && apt-get -y install \
 	bash \
 	expect \
 	jq \
-	curl
+	curl \
+	vim \
+	git
```

Comment:
This is a pretty good idea, and something we should maybe add to the other Dockerfiles as well :heavy_check_mark: